FF8.Core/Sources/WalkmeshCoords.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+
+namespace FF8.Core
+{
+    public struct WalkmeshCoords
+    {
+        public readonly Int32 TriangleId;
+        public readonly Int32 X;
+        public readonly Int32 Y;
+        public readonly Int32 Z;
+        public readonly Boolean HasZ;
+
+        public WalkmeshCoords(Int32 triangleIdId, Int32 x, Int32 y)
+        {
+            TriangleId = triangleIdId;
+            X = x;
```

Comment:
_same as coord- coordinate system may be different. Needs testing in-game_

Reply:
there's cords where the 3 characters stand in the save file as well. though it's x and y and are uint16.